server/controllers/usersController.js

```diff
@@ -26,6 +32,55 @@ class Registered {
     });
   }
 
+//Db controllers
+async create(req, res) {
+  const text = `INSERT INTO users
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)
+    RETURNING *`;
+  const {
+    firstName, lastName, email, password, address, bio, occupation, expertise,
+  } = req.body;
+  const hashPassword = Helper.hashPassword(password);
+  const values = [
+    uuid(),
+    firstName,
+    lastName,
+    email,
+    hashPassword,
+    address,
+    bio,
+    occupation,
+    expertise,
+    moment(new Date()),
+    moment(new Date()),
+  ];
+  try {
+    if (!Helper.isValidEmail(req.body.email)) {
```

Comment:
Your controller is doing a lot of work. Make use of a middleware and then use it in the route
e.g.

` router.get('/auth/signup', validateUser, userController); `

The validateUSer method will be called first and if you place the `next()` method call in that method then userController is called next